docker/Dockerfile.pgi

```diff
@@ -43,7 +43,7 @@ RUN CMAKE_VERSION=3.13.4 && \
     grep ${CMAKE_SCRIPT} ${CMAKE_SHA256} | sha256sum --check && \
     mkdir -p ${CMAKE_DIR} && \
     sh ${CMAKE_SCRIPT} --skip-license --prefix=${CMAKE_DIR} && \
-    rm ${CMAKE_SCRIPT}
+    rm cmake*
```

Comment:
Do you want `cmake*` as here or `cmake-*` as above?

Reply:
cmake*